extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/DisabledSchedulerTest.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DisabledSchedulerTest {
+
+    @Inject
+    Scheduler quartzScheduler;
```

Comment:
```suggestion
    Scheduler scheduler;
```

Reply:
Also this tests is really like this one https://github.com/quarkusio/quarkus/pull/9027/files#diff-5028b2d8d0d2b040363a8c0981f36caeR17, I think we do not need this.

Reply:
Not sure, because this class tests the QuartzScheduler (org.quartz.Scheduler) functionality. While the other test is about SimpleScheduler functionality.

Reply:
Yes, except I am not seeing any quartz related stuff being tested. I think we can remove this test class WDYT? @mkouba. 

Other than that this PR is good to go once all the commits have been squashed.

Reply:
> Yes, except I am not seeing any quartz related stuff being tested. I think we can remove this test class...

I'd rather keep the test because the injected `io.quarkus.scheduler.Scheduler` is the `io.quarkus.quartz.runtime.QuartzScheduler`. So it tests our quartz integration.

Reply:
Sorry, I confused it with quartz specific scheduler. Let’s keep it then.